ui/src/app/job-details/panels/panels.component.ts

```diff
@@ -135,6 +137,25 @@ export class JobPanelsComponent implements OnInit {
     return !this.hasParent() && (this.job.status == JobStatus.Submitted || this.job.status == JobStatus.Running);
   }
 
+  copyJobIdToClipboard(): void {
+    try {
+      const jobIdInput = document.querySelector('#job-id') as HTMLInputElement;
+      jobIdInput.select();
+      document.execCommand('copy');
+      this.changeCopyIcon('check');
+    } catch (error) {
+      this.changeCopyIcon('times');
+      console.log(error);
+    }
+  }
+
+  changeCopyIcon(newIcon: string): void {
+    this.copyIcon = newIcon;
+    setTimeout(() => {
```

Comment:
Is there a possible scenario where things go wrong if someone clicks it again within 1.5 seconds?

Reply:
I clicked the button several times in quick succession and saw no issues.